server/src/main/java/org/elasticsearch/gateway/GatewayMetaState.java

```diff
@@ -230,7 +224,7 @@ public static Metadata upgradeMetadata(Metadata metadata,
         // upgrade current templates
         if (applyPluginUpgraders(
             metadata.templates(),
-            metadataUpgrader.indexTemplateMetadataUpgraders,
+            metadataUpgradeService::upgradeTemplates,
```

Comment:
I think `applyPluginUpgraders` should be renamed to `applyUpgraders`, given the point of the first commit

Reply:
Renamed it to `applyUpgraders`